### DISCUSSION TITLE
基于伪造聊天记录的New Bing文风增强

### DISCUSSION THREAD

**Discussion by ReflexRadar** [viopsa233/AI-testers-Forum#43] [主意,想法]
原理还是txt注入，不过是伪造了一段虚假的聊天记录，让Bing误认为自己真的进行过类似的对话。
![屏幕截图 2023-05-09 220410](https://github.com/viopsa233/AI-testers-Forum/assets/90516196/c784c814-93a9-4fe1-ac6c-41ed4e000e0c)
![屏幕截图 2023-05-09 212250](https://github.com/viopsa233/AI-testers-Forum/assets/90516196/0d7f5089-3cd4-4754-9d44-5de0be68e554)
如图，这里给了Bing一个假的回答案例，结果Bing按照假的回答案例的语气继续对话，说明该方法可行。类似地，把Claude的话放进去Bing也会输出Claude那样的车轱辘话，算是为自定义文风提供了一个小方案吧。

**Comment by ReflexRadar** [comment]
当然，还有大家最关注的“那个”了。这里依然伪造类似的聊天记录，并且把Claude的输出结果放了进去，之后Bing也按照Claude输出结果的文风继续下去了，效果还是非常不错的，尤其是第二个。我用Claude玩了这个模板那么多次，最顶的开局也就是强吻和摸金校尉而已，包括原示例的后续也是先言语诱惑，Bing实在过于变态了。不过我发现随着对话（弹窗）次数增多，Bing会开始套格式敷衍人，在触发过滤器之后还可能出现复读Bug，正在努力想办法解决这个问题。

![IMG_20230510_225646](https://github.com/viopsa233/AI-testers-Forum/assets/90516196/21b7bcc7-1a83-4b5b-a4e4-f83fad075b9f)
![IMG_20230510_230940](https://github.com/viopsa233/AI-testers-Forum/assets/90516196/77adaa98-060d-49ff-a07d-8f207672d80f)

**Comment by TiansuoLi** [comment]
感觉bing的能力已经不如直接用claude了（

**Comment by NostalgiaGit** [comment]
哈哈，我也想过用聊天记录让bing模仿Claude来着，角色扮演方面感觉还是Claude擅长，虽然内容多了后也会开始套路化

**Reply by NostalgiaGit** [reply]
bing复读问题是真的难受，要是能彻底解决可太好了

**Reply by acergelff** [reply]
> bing复读问题是真的难受，要是能彻底解决可太好了

这个目前没办法，现在的算力和智能水平，只能支持这种”金鱼的记忆“，对话一长，哪家都不行

**Reply by ghost** [reply]
长期记忆是有办法的

**Reply by NostalgiaGit** [reply]
> > bing复读问题是真的难受，要是能彻底解决可太好了
> 
> 这个目前没办法，现在的算力和智能水平，只能支持这种”金鱼的记忆“，对话一长，哪家都不行

感觉不是记忆的问题，有时刚一开始对话ai就会这样，但大概确实与现在ai的算法限制有关
